sklearn/metrics/ranking.py

```diff
@@ -240,8 +241,9 @@ def _binary_uninterpolated_average_precision(
                                  average, sample_weight=sample_weight)
 
 
-def roc_auc_score(y_true, y_score, average="macro", sample_weight=None,
-                  max_fpr=None):
+def roc_auc_score(y_true, y_score, labels=None,
+                  multiclass="ovr", average="macro",
```

Comment:
I've realised that users will pass binary y_true and 2d y_score as output from predict_proba of a binary classifier. Am I right to think the proposed implementation will treat this as multiclass? Will they get a different result to just passing the second column of predict_proba or is the result the same? Have we considered this before? (Not thinking clearly right now)

Reply:
If `y_true` is binary, and `y_score` is 2d, this PR does not change the behavior on master. (It raises a `ValueError`)

In the docs, for `roc_auc_score` under `y_score` we have:

> For binary y_true, y_score is supposed to be the score of the class with greater label.